### DISCUSSION TITLE
Звук двигателя в кабине [Cockpit engine sound]

### DISCUSSION THREAD

**Discussion by yuriks59** [flybywiresim/aircraft#2314] [Help] [answered]
Во время полета звук в кабине настолько громкий, что после часа полета голова болит от гула. На внешней камере звук тихий, а в кабине громко, как будто на двигателях кабина стоит. Реально кабина имеет хорошую звукоизоляцию и удалена лот двигателей, что снижает шум в кабине. И в настройках уменьшить громкость невозможно.

**Comment by hotshotp** [comment, marked as answer]
Hi, please write in english next time as most of the people in this community speak english. I think it's not the engines that are too loud for you but the wind. That was because of a mistake I made in an update a few days back and has now been fixed. Please download the latest development version which will fix your issue.

**Reply by yuriks59** [reply]
Thank you, now I will install the latest version, let's see the result.

**Comment by yuriks59** [comment]
During the flight, the sound in the cockpit is so loud that after an hour of flight, the head hurts from the hum. On the external camera, the sound is quiet, but in the cabin it is loud, as if the cabin is on engines. In reality, the cabin has good sound insulation and is removed from the engines, which reduces noise in the cabin. And you can't turn down the volume in the settings.
